contracts/RelayHub.sol

```diff
@@ -159,20 +161,25 @@ contract RelayHub is RelayHubApi {
         bytes32 hashed_message = keccak256(abi.encodePacked(packed, relay));
         bytes32 signed_message = keccak256(abi.encodePacked("\x19Ethereum Signed Message:\n32", hashed_message));
         if (!GsnUtils.checkSig(from, signed_message,  approval))  // Verify the sender's signature on the transaction
-            return 1;   // @from hasn't signed the transaction properly
+            return uint32(CanRelayStatus.WrongSignature);   // @from hasn't signed the transaction properly
```

Comment:
we can change this `uint32` to `uint`. the only reason its `uint32` is my mistake: it looked more "native" to me - which it isn't.... There's no reason to limit it to 32bit.

Reply:
I agree, I just think we should do that as part of a bigger refactoring pull request (that will also have snake to camel case changes etc.).